app/views/govuk_component/docs/breadcrumbs.yml

```diff
@@ -12,6 +15,19 @@ examples:
         url: '/section'
       - title: 'Sub-section'
         url: '/section/sub-section'
+  inverse:
+    description: On a dark background
```

Comment:
It's useful to say where this will be used in the description, e.g: on topic pages. When the topic pages go live, we can then link to a live page here so the component can be seen in context.